src-tauri/src/plugins/cohdb/auth/mod.rs

```diff
@@ -96,7 +96,7 @@ async fn authenticate<R: Runtime>(handle: AppHandle<R>) -> Result<String> {
     *state.request.lock().await = Some(request);
 
     info!("redirecting to auth URL: {auth_url}");
-    open(&handle.shell_scope(), auth_url.clone(), None).map_err(Shell);
+    open(&handle.shell_scope(), auth_url.clone(), None).map_err(Shell)?;
```

Comment:
The `?` here just makes sure any error halts execution propagates up to the caller (without it the error is swallowed).

This call really shouldn't ever fail though so it probably doesn't actually make a difference here.